### ISSUE TITLE
Bad custom schema attributes

### ISSUE THREAD

**Issue by konohomaru** [MightyOrm/Mighty#44]
Is it possible switch from custom schema mapping attributes such as `DatabaseTable`, `DatabaseColumn` to default attributes from `System.ComponentModel.DataAnnotations.Schema`?

**Comment by mikebeaton** [comment]
Hmmm I will look at this - thanks for the suggestion.

There are ofc multiple options for schema attributes, at the time I settled on the solution which seemed most 'Mighty-like' to me (!), resonably nice, accessible syntax but powerful enough for most reasonable use-cases.

Perhaps you could confirm the motivation, I assume just to be more 'standard', and compatible with something people are likely to be used to or to use elsewhere already? (And/or to work with other (micro-)ORMs at the same time?)

**Comment by konohomaru** [comment]
There are two layers in my solution: domain and server. In the domain layer there is only the business logic of my application, in the server layer there are all technical solutions. MightyOrm in my case is also a technical solution and is hidden in the server layer. However, due to custom mapping attributes, my domain layer still depends on MightyOrm.

**Comment by mikebeaton** [comment]
I mean, just thinking about it, you would ideally like to get the database layer code not mentioned in the the domain layer. But `System.ComponentModel.DataAnnotations.Schema` _is_ database layer code. So you'd just prefer to use MS standard database layer annotations, not database layer annotations which are specific to the technical solution you are using. Does that sound correct?

**Comment by konohomaru** [comment]
Yes, you are right. But I realize another side of this question. That's when we can use default tools it is good practice to use default tools. Because more people are familiar with it. Also some custom attributes looks like default is not enough and the question is why.
